DAQ_FW/src/main.cpp

```diff
@@ -1,12 +1,19 @@
 #include <Arduino.h>
+#include "Log.h"
+using namespace Logger;
 
-void setup() {
+void setup()
+{
   // put your setup code here, to run once:
   Serial.begin(9600);
```

Comment:
Since I think your Logger module uses Serial and every other piece of code shouldn't be using Serial functions, it makes sense for Logger to "own" the Serial module and be responsible for any initialization, etc. If that's the case then this should go inside `Logger::Start()` 

Also, the 9600 baud rate can be a constant inside the `system_config.h` file I talked about earlier.